docs/data/material/migration/migrating-to-v6/migrating-to-v6.md

```diff
@@ -186,6 +186,10 @@ export default function ChipExample() {
 }
 ```
 
+### Divider
+
+When using vertical orientation, the Divider now renders a `<div>` with the corresponding accessibility attributes instead of `<hr>` to adhere to the WAI-ARIA [spec](https://www.w3.org/TR/wai-aria-1.2/#separator). You might need to adjust your styles accordingly if you are targeting `hr` tags in your CSS.
```

Comment:
```suggestion
When using vertical orientation, the Divider now renders a `<div>` with the corresponding accessibility attributes instead of `<hr>` to adhere to [the WAI-ARIA spec](https://www.w3.org/TR/wai-aria-1.2/#separator). You might need to adjust your styles accordingly if you are targeting `hr` tags in your CSS.
```
I remember that @samuelsycamore suggest to use 2-3 words for a link

Reply:
Updated